docs/requirement-sets/excel/custom-functions-requirement-sets.md

```diff
@@ -12,13 +12,14 @@ ms.localizationpriority: medium
 
 | Requirement set | Office on Windows<br>- Microsoft 365 subscription<br>- retail perpetual Office 2016 and later | Office on Windows<br>(volume-licensed perpetual) | Office on Mac | Office on iPad | Office on the web |
 |:-----|:-----|:-----|:-----|:-----|:-----|
+| CustomFunctionsRuntime 1.4 | Version 2208 (Build 15601.20148) | Office 2021: Version 2208 (Build 15601.20148) | 16.64 | Not supported | Supported |
```

Comment:
I don't think VL Office 2021 supports this reqt. set. Should likely be supported in Office vNext
```suggestion
| CustomFunctionsRuntime 1.4 | Version 2208 (Build 15601.20148) | Not supported | 16.64 | Not supported | Supported |
```